seatunnel-engine/seatunnel-engine-storage/imap-storage-api/src/main/java/org/apache/seatunnel/engine/imap/storage/api/common/ProtoStuffSerializer.java

```diff
@@ -77,6 +77,7 @@ private static <T> Schema<T> getSchema(Class<T> clazz) {
         WRAPPERS.add(TreeMap.class);
         WRAPPERS.add(Hashtable.class);
         WRAPPERS.add(SortedMap.class);
+        WRAPPERS.add(Long[].class);
```

Comment:
Need to support arrays of other primitive types? e.g.(byte[]、short[]、int[]、boolean[]、float[]、double[]、String[])

Reply:
Usually we do not suggest the array type in serialization, if necessary, it is recommended to use List.

Reply:
I remember that arrays are problematic to serialize in some special cases. But I can't find the corresponding documentation.

Reply:
> Also, I think this is more of an enhancement than a bug.

@liugddx  update this pr title: [Improve] xxx...

Reply:
> elements
I have provided the test cases and they have passed.
![image](https://user-images.githubusercontent.com/48236177/206954837-d3ce32cb-7724-4620-8e60-5c6a2452424e.png)

Reply:
![image](https://user-images.githubusercontent.com/16631152/206959434-40107ad4-e328-4ed8-8726-70191d9448d1.png)

Reply:
> ![image](https://user-images.githubusercontent.com/16631152/206959434-40107ad4-e328-4ed8-8726-70191d9448d1.png)

![image](https://user-images.githubusercontent.com/48236177/206960152-0aa7fe0e-2bf0-4338-897e-9bd02fca5932.png)
Like this？

Reply:
> ![image](https://user-images.githubusercontent.com/16631152/206959434-40107ad4-e328-4ed8-8726-70191d9448d1.png)

![image](https://user-images.githubusercontent.com/48236177/206960334-29b12132-753f-4a50-b867-7d32a7299c11.png)